src/main/kotlin/com/sourcegraph/cody/chat/ui/ChatPanel.kt

```diff
@@ -47,10 +51,31 @@ class ChatPanel(project: Project, val chatSession: ChatSession) :
     add(lowerPanel, BorderLayout.SOUTH)
   }
 
+  fun addModelDropdown(project: Project, sessionId: String, selectedModel: String) {
+    CodyAgentService.applyAgentOnBackgroundThread(project) { agent ->
+      agent.server.isCurrentUserPro().thenApply { isUserPro ->
```

Comment:
Can we change that `getIsCurrentUserPro` function to return `Boolean` and not `Boolean`?
We can just add `?: false` at the end of it.
Then `if (isUserPro == true)` will turn into `if (isUserPro)` here and in other places where we use it.